must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -11,6 +11,9 @@ if [ "${BASE_COLLECTION_PATH}" = "" ]; then
     BASE_COLLECTION_PATH=$(pwd)
 fi
 
+# Make a global variable for namespace
+INSTALL_NAMESPACE=openshift-storage
```

Comment:
...I'm just gonna say that I don't like this, we should be getting this from an env var or something, but I'm not going to ask for this to change. The noobaa gather script is a mess in many other ways and needs a broader refactor, which would prevent this.

Reply:
Agreed, I will refactor the code and make it a bit cleaner after this PR